src/targets/gpu/include/migraphx/gpu/gemm.hpp

```diff
@@ -19,6 +19,7 @@ template <class Op>
 struct rocblas_gemm
 {
     Op op;
+    bool int8_x4_format = true;
```

Comment:
This needs to be added to `reflect` so it will be saved. Most likely it will need to be `pack_join(migraphx::reflect(self.op, f), pack(f(self.int8_x4_format, "int8_x4_format")))`